client/question/components/PrivacyModalDirective.js

```diff
@@ -0,0 +1,159 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the privacy agreement modal.
+ */
+
+tie.directive('privacyModal', [function() {
+  return {
+    restrict: 'E',
+    scope: {
+      isDisplayed: '=',
+      title: '@',
+      isClientVersion: '&'
```

Comment:
Can we get rid of this binding and just depend on [ServerHandlerService](https://github.com/google/tie/blob/master/client/question/services/ServerHandlerService.js)? (Same thing in the learner view.)